packages/react/src/FronteggProvider.tsx

```diff
@@ -8,6 +8,7 @@ import { AppHolder } from '@frontegg/js/AppHolder';
 import { useQueryKeeper } from './queryKeeper';
 import { isAuthRoute } from '@frontegg/redux-store';
 import sdkVersion from './sdkVersion';
+import ReactPkg from 'react/package.json'
```

Comment:
no need for the loader you added in other wrappers?

Reply:
nope